include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -551,8 +551,7 @@ namespace internal
             for (auto cell_id : i.second)
               {
                 typename MeshType::cell_iterator cell(
-                  *tria_dst.create_cell_iterator(
-                    cell_id_translator.to_cell_id(cell_id)),
```

Comment:
CellId::to_cell is deprecated. I think you can safely keep the old code here and in all the other occasions.